plugins/source/aws/resources/services/frauddetector/rules_fetch.go

```diff
@@ -0,0 +1,32 @@
+package frauddetector
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/frauddetector"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchFrauddetectorRules(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+	svc := c.Services().FraudDetector
+
+	input := new(frauddetector.GetRulesInput)
+	for {
+		output, err := svc.GetRules(ctx, input)
```

Comment:
This type (and most of the others, by the looks of it) supports a paginator: https://docs.aws.amazon.com/sdk-for-go/api/service/frauddetector/#FraudDetector.GetRulesPages I think we should use that, since it's a bit simpler and there's less scope for bugs if we don't need to handle the pagination tokens in our code.

Reply:
Will look into that, thanks!

Reply:
done